deployability/modules/allocation/README.MD

```diff
@@ -0,0 +1,444 @@
+## Allocation Module
+
+### User documentation
+
+The Allocation module allows you to create and destroy VMs both locally and in AWS, it also gives the possibility of creating macOS VMs on Wazuh infrastructure or PowerPC VMs, as long as you have the necessary permissions to do so. VMs can be AMD64, ARM64, Windows, macOS (Intel and ARM), and PowerPC (Centos and Debian).
+
+#### Set the environment
+
+The execution of the allocation is carried out through the Workflow library, or by executing them manually through commands.
+Execution can be done from any operating system.
+
+Initially, You must install Python libraries. We recommend to use virtual environments. Follow the technical documentation at https://docs.python.org/3/library/venv.html.
+
+1. Activate the environment:
+
+  ```bash
+  source {venv directory}/bin/activate
+  ```
+
+2. Clone the `wazuh-qa` repository:
+
+  Navigate to the project directory and switch to the project branch:
+
+  ```bash
+  cd wazuh-qa
+  git checkout {project-branch}
+  ```
+> Note: temporary dev project-branch is [enhancement/4495-DTT1](https://github.com/wazuh/wazuh-qa/tree/enhancement/4495-DTT1)
+
+3. Install requirements:
+
+  ```bash
+  pip3 install -r deployability/deps/requirements.txt
+  ```
+
+#### Use the Allocation module through the Workflow
+
+Now, it is possible to use the worklow engine library to launch the provision module by doing the following steps:
+
+1. Install the Workflow engine library and its launcher:
+
+  While in wazuh-qa:
+
+  ```bash
+  cd modules
+  pip3 uninstall -y workflow_engine && pip3 install .
+  ```
+
+2. Test Fixture to Execute:
+
+      It will be necessary to create a fixture (yaml file) where the infrastructure, provisioning, and tests to be executed will be declared.
+
+      >Note: It is possible to find some fixture examples in [deployability/modules/workflow_engine/examples/](../workflow_engine/examples)
+
+      Example:
+
+      ```bash
+      version: 0.1
+      description: This workflow is used to test agents' deployment for DDT1 PoC
+      variables:
```

Comment:
I don't know if we should add an example using other modules, or at least, make focus on the Allocation module more than the testing module

Reply:
Done ee033425e